echo/server/dembrane/api/stateless.py

```
@@ -228,7 +228,7 @@ async def get_lightrag_prompt(payload: GetLightragQueryRequest,
                                ids= [str(id) for id in echo_segment_ids],
                                top_k = payload.top_k)
             response = await rag.aquery(payload.query, param=param)
-            logger.debug(f"***Response: {response}")
+            logger.debug(f"Response: {response}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)